index.html

```diff
@@ -0,0 +1,135 @@
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
+        </div>
+        <div class="card">
+            <div class="works-heading"></div>
+            <div class="works-footer">
+                <p class="Project-title">Multi-Post stories Gain+Glory</p>
+                    <ul class="list-inline">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>Javascript</li>
+                        <li>html</li>
+                    </ul>
+                 <input type="submit" value="See Project" class="btn"/>
+            </div>
```

Comment:
- [ ] Kindly wrap this content with a section tag for better semantic HTML, and replace div with a section tag.